easybuild/easyconfigs/f/FastME/FastME-2.1.6.1-GCC-10.2.0.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'FastME'
+version = '2.1.6.1'
+
+homepage = 'http://www.atgc-montpellier.fr/fastme/'
+description = "FastME: a comprehensive, accurate and fast distance-based phylogeny inference program."
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = ['https://gite.lirmm.fr/atgc/FastME/-/archive/v%(version)s/']
+sources = ['FastME-v%(version)s.tar.gz']
+checksums = ['fae0609576873cb137ad4f63529b3ccba8f747b857c23c7870236aba440076ca']
+
+builddependencies = [
+    ('binutils', '2.35'),
```

Comment:
@boegel binutils is already included in GCC toolchain?

Reply:
```suggestion
```
as one easy click